magento/magento2
# Fix for #14208: Order of catalog images not respected on store view level

### Description (#14208)
Order of catalog images not respected on store view level

### Preconditions

    Magento 2.2.2

### Linked issues:

magento/community-features#23
Steps to reproduce

    1. Go to product that has multiple images
    2. set scope to all storeviews
    3. change the first image in the gallery
    4. save the product
    5. switch to store view scope (notice that the image order is correct, for now)
    6. save the product
    7. switch back to all store views
    8. change the order of the images again
    9. save the product
    10. go back to storeview scope
    11. notice that the order of the images is not correct anymore.

There should be some sort of 'use default value' checkbox for the media gallery, or at least for the position.
#### Expected result

    The image positions should be according the 'All store views'

#### Actual result

    The image positions use there own position, copied from the 'all storeviews', and don't change anymore according to the 'all store views'


### Fixed Issues
1. magento/magento2#14208: Order of catalog images not respected on store view level

### Manual testing scenarios
Steps to reproduce
   1. Go to product that has multiple images
   2. set scope to all storeviews
   3. change the first image in the gallery
   4. save the product
   5. switch to store view scope (notice that the image order is correct, for now)
   6. save the product
   7. switch back to all store views
   8. change the order of the images again
   9. save the product
   10. go back to storeview scope
   11. notice that the order of the images is not correct anymore.

### Contribution checklist (*)
